src/lib/y2storage/proposal/partitions_distribution_calculator.rb

```diff
@@ -247,7 +247,7 @@ def partition_candidate_spaces(partition, candidate_spaces, extra_spaces)
       #
       # @return [Boolean]
       def compatible_disk?(partition, space)
-        return true unless partition.disk && partition.disk != space.disk_name
+        true unless partition.disk && partition.disk != space.disk_name
```

Comment:
It says `@return [Boolean]` but it actually returns `true` or `nil`... :thinking: 

Yard strictly defines `Boolean` as either `true` or `false` values, nothing else.